pro/src/routes/CollectiveOfferRoutes/CollectiveOfferRoutes.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { useParams, Route, Switch } from 'react-router-dom'
+
+import { extractOfferIdAndOfferTypeFromRouteParams } from 'core/OfferEducational'
+
+import CollectiveOfferCreationFromTemplateRoutes from './CollectiveOfferCreationFromTemplateRoutes/CollectiveOfferCreationFromTemplateRoutes'
+import CollectiveOfferCreationRoutes from './CollectiveOfferCreationRoutes'
+import CollectiveOfferEditionRoutes from './CollectiveOfferEditionRoutes'
+import CollectiveOfferTemplateEditionRoutes from './CollectiveOfferTemplateEditionRoutes'
+
+const CollectiveOfferRoutes = (): JSX.Element => {
+  const { offerId: offerIdFromParams } = useParams<{
+    offerId: string
+  }>()
+
+  const { offerId, isShowcase } =
+    extractOfferIdAndOfferTypeFromRouteParams(offerIdFromParams)
+
+  const formattedOfferId = offerId !== 'creation' ? offerId : undefined
+
+  return (
+    <Switch>
+      <Route
+        path={[
+          '/offre/duplication/collectif/:offerId/stocks',
+          '/offre/duplication/collectif/:offerId/visibilite',
+          '/offre/duplication/collectif/:offerId/confirmation',
+          '/offre/duplication/:templateId',
```

Comment:
proposition: 
dans route_maps: 
```js
path: '/offer/collective',
exact: false,
component: CollectiveOfferRoutes
```

ici: 
```suggestion
          'duplication'
```
on pourrait acceder aux pages via: 
* `/offer/:offerId/collective/duplication/stocks`
* `/offer/:offerId/collective/duplication/visibilite`
* `/offer/:offerId/collective/duplication/confirmation`
* `/offer/:offerId/collective/duplication/:templateId`

Voir à effacer ce composant pour directement avoir dans route_map: 
```js
path: '/offer/collective/duplication',
exact: false,
component: CollectiveOfferCreationFromTemplateRoutes
```

Reply:
J'ai pas tout compris je veux bien que tu détailles un peu plus stp ?
Pour la dernière route (celle que tu proposes étant `/offer/:offerId/collective/duplication/:templateId`) on n'a pas d'`offerId` à cette étape là car on n'a pas encore créé l'offre du coup les URLs seraient de toute facon pas les même :/